typings/capitano/index.d.ts

```diff
@@ -78,4 +80,14 @@ declare module 'capitano' {
 		commands: Command[];
 		globalOptions: OptionDefinition[];
 	};
+
+	export function execute(
+		args: any,
+		callback: (err?: any, result: any) => void,
+	): void;
```

Comment:
This could probably be typed better but I figure it's not worth it with the oclif push